privacyidea/api/lib/prepolicy.py

```diff
@@ -153,6 +153,54 @@ def policy_wrapper(*args, **kwds):
         return policy_wrapper
 
 
+def generate_pin_from_policy(policy, size=6):
```

Comment:
As far as I understand, this is a module specific function which is not used as a decorater, but only within this module.
So please rename it to ``_generate_pin_from_policy``.

Reply:
Please add some tests for this method, to see, if pins are generated in the correct way.

Reply:
tests added